google/cloud/bigquerycontrol/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# ~~~
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+# The tests require googletest to be installed. Force CMake to use the config
+# file for googletest (that is, the CMake file installed by googletest itself),
+# because the generic `FindGTest` module does not define the GTest::gmock
+# target, and the target names are also weird.
+find_package(GTest CONFIG REQUIRED)
```

Comment:
We have a helper for this:

```
include(FindGMockWithTargets)
```

We are still a few CMake revisions away from not needed that helper:

https://cmake.org/cmake/help/latest/module/FindGTest.html#imported-targets